src/js/control-bar/text-track-controls/descriptions-button.js

```diff
@@ -0,0 +1,25 @@
+import CaptionsButton from './captions-button.js';
+
+/**
+ * The button component for toggling and selecting descriptions
+ *
+ * @constructor
+ */
+class DescriptionsButton extends CaptionsButton {
```

Comment:
Is there a reason this needs subclass CaptionsButton instead of TextTrackButton?

If I were to guess it would be because there doesn't appear to be any handling of native vs non-native tracks in TextTrackButton. @gkatsev is there a reason why that's in CaptionsButton and not TextTrackButton?